.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: test-pull-request
```

Comment:
Probably change that to `Run unit tests` or some similarly descriptive name. That's the name that shows up in the UI.

Reply:
Actually, probably just `CI tests` so we can put more stuff in `jobs` for other aspects like static analysis and whatnot.

Reply:
Done